pkg/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const ID_LENGTH = 16;
 export const ENCRYPTION_KEY_LENGTH = 128;
+export const LATEST_KEY_VERSION = 2;
```

Comment:
Moved this to constants since now this is important for the API endpoint to encrypt/decrypt similarly to the front end.

Reply:
Good call